<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>668</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>691</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>266</width>
     <height>627</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>266</width>
     <height>627</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>OCR-B 10 BT</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="Saho" country="Eritrea"/>
   </property>
   <widget class="QLabel" name="login_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR-A BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter your login name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR-A BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter your password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="login_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR-A BT</family>
      <pointsize>16</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>alshfu13</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_value">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR-B 10 BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>as785ghqw590</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_name_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR-A BT</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter range betwen min and max folwers:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_folowers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR-B 10 BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_folowers">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR-B 10 BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="run">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR-B 10 BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="min_word_value">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR-B 10 BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="login_name_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR-A BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter min word 
 in comments</string>
    </property>
   </widget>
   <widget class="QLabel" name="info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="min_word_value_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR-B 10 BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="login_name_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR-A BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter min posts 
 for parsing</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>266</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
